src/stores/torrent-store.js

```diff
@@ -22,13 +22,11 @@ const extractDomains = (torrent) => {
 class TorrentStore {
   @observable previousTorrents = [];
   @observable torrents = [];
-  @observable statusFilter = 0;
   @observable trackerFilter = '';
   @observable textFilter = '';
-  @observable sortCriteria = 'name';
-  @observable sortDirection = '';
 
-  constructor(rpc) {
+  constructor(rpc, prefs_store) {
```

Comment:
This is how I planned on doing this too initially, as this is how I'd do it with ie Relay/Redux,, however I'm not sure of the implications of the computed filteredTorrents - I'm guessing it's an essentially memoized property when a store value is changed. (first foray with MobX)

So I guess my question is just to confirm it'll be best to move the filteredTorrents function to the component @edulan?